assets/styles/about_us.scss

```diff
@@ -0,0 +1,307 @@
+    font-size: 1.125rem;
+    font-style: normal;
+    font-weight: 400;
+    line-height: normal;
+}
+
+.leftSide {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    grid-column: 1/2;
+    grid-row: 1;
+}
+
+.profil {
+    display: flex;
+    justify-content: center;
+    align-items: start;
+}
+
+article .profil img {
+
+    max-width: 18.75rem;
+    max-height: 18.75rem;
+    width: 100%;
+    border-radius: 50%;
+
+}
+
+article .firstquote {
+    margin-right: 0.5rem;
+    grid-column: 2/3;
+}
+
+article aside {
+    grid-column: 3/5;
+    grid-row: 1/3;
+}
+
+article aside p span {
+    flex-direction: row-reverse;
+    display: flex;
+}
+
+.authorName {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.authorName h2 {
+    color: var(--our-black);
+    font-family: Titillium Web;
+    font-size: 1rem;
+    font-style: normal;
+    text-align: center;
+    font-weight: 700;
+    line-height: normal;
+    margin-top: 1rem;
+}
+
+.website {
+    margin-top: 1rem;
+    display: flex;
+    justify-content: center;
+    align-items: end;
+}
+
+.website a {
+    justify-content: center;
+    align-items: center;
+    text-decoration: inherit;
+    color: var(--our-black);
+    display: flex;
+}
+
+.website a:hover {
+    -webkit-animation-name: pulse;
+    animation-name: pulse;
+    -webkit-animation-duration: 1s;
+    animation-duration: 1s;
+    -webkit-animation-fill-mode: both;
+    animation-fill-mode: both;
+}
+
+.linkedinProfil {
+    grid-row: 3/4;
+    grid-column: 3/4;
+    display: flex;
+    justify-content: center;
+    text-decoration: inherit;
+    color: var(--our-black);
+    border-radius: 0.625rem;
+    background: var(--main-color);
+    box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+    max-width: 255px;
+    min-width: 190px;
+    width: 90%;
+    max-height: 53px;
+    flex-shrink: 0;
+    align-items: center;
+    margin-right: 1rem;
+}
+
+.linkedinProfil:hover {
+    -webkit-animation-name: pulse;
+    animation-name: pulse;
+    -webkit-animation-duration: 1s;
+    animation-duration: 1s;
+    -webkit-animation-fill-mode: both;
+    animation-fill-mode: both;
+}
+
+.githubProfil {
+    grid-row: 3/4;
+    grid-column: 4/5;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: inherit;
+    color: var(--our-black);
+    max-width: 255px;
+    min-width: 190px;
+    width: 90%;
+    max-height: 53px;
+    border-radius: 0.625rem;
+    border: 1px solid var(--main-color);
+    background: rgba(255, 164, 88, 0.07);
+    box-shadow: 0px 4px 4px 0px rgba(0, 0, 0, 0.25);
+}
+
+.githubProfil:hover {
+    -webkit-animation-name: pulse;
+    animation-name: pulse;
+    -webkit-animation-duration: 1s;
+    animation-duration: 1s;
+    -webkit-animation-fill-mode: both;
+    animation-fill-mode: both;
+}
+
+article a img {
+    margin-right: 1rem;
+}
+
+/* Mobile */
+
+@media screen and (max-width:1100px) {
+    main {
```

Comment:
is it a global behavior for all main of all pages ? shift it into global css file